docs/user/adaptive-learning.rst

```diff
@@ -64,6 +67,35 @@ Alternatively, instructors can also link competencies to an exercise or lecture
 
 |instructors-learning-goals-link|
 
+Learning Paths
+--------------
+
+Learning Paths for Students
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Students can access their learning path in the learning path tab. Here they are able to access recommended lecture units and participate in exercises.
```

Comment:
Missing comma after "Here they are..." --> "Here, they are..."
I would change "Here they are able to... " --> "Here, they can ..."